README.md

```
@@ -402,6 +406,273 @@ revisium rows upload --folder ./data --commit
+revisium://admin:secret@cloud.revisium.io/org/proj/master:head
+
+# Credentials via environment variables
+export REVISIUM_SOURCE_USERNAME=admin
+export REVISIUM_SOURCE_PASSWORD=secret
+```
+
+**Interactive Mode:**
+
+If no credentials are provided, you'll be prompted to choose:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)